doc/advopt.txt

```diff
@@ -149,11 +145,8 @@ Advanced options:
   --legacy:$2
                             enable obsolete/legacy language feature
   --useVersion:1.0|1.2      emulate Nim version X of the Nim compiler, for testing
-  --profiler:on|off         enable profiling; requires `import nimprof`, and
```

Comment:
@Araq why remove this?

also, unsupported options should IMO trigger warnings

Reply:
Nimprof's output is usually incomprehensible and misleading and nimprof has no maintainer.

Reply:
> has no maintainer

Well, I [tried](https://github.com/nim-lang/Nim/pull/10119)

Reply:
> Profiling is important though, we should offer a solution that's reliable, even if its' just wrapping around some platform specific standard tool.

Exactly, we need to delegate to a separate tool.